content/en/docs/concepts/security/index.md

```diff
@@ -12,565 +12,546 @@ aliases:
-
-1. The Istio Agent sends both the certificate received from Citadel and the
-   private key to Envoy.
-
-1. The above CSR process repeats periodically for certificate and key rotation.
-
-### The Istio Agent in Kubernetes
-
-Istio provides the option of using the Istio Agent in Kubernetes for certificate and key provisioning, as shown in the figure below.
-Note that the identity provisioning flow for on-premises machines will be similar in the near future, we only describe the Kubernetes scenario here.
-
-{{< image width="80%" link="./node_agent.svg" caption="PKI with the Istio Agent in Kubernetes"  >}}
-
-The flow goes as follows:
-
-1. Citadel creates a gRPC service to take CSR requests.
-
-1. Envoy sends a certificate and key request via Envoy secret discovery service (SDS) API.
-
-1. Upon receiving the SDS request, the Istio Agent creates the private key and CSR before sending the CSR with its credentials to Citadel for signing.
-
-1. Citadel validates the credentials carried in the CSR and signs the CSR to generate the certificate.
-
-1. The Istio Agent sends the certificate received from Citadel and the private key to Envoy via the Envoy SDS API.
+Identity is a fundamental concept of any security infrastructure. At the
+beginning of a workload-to-workload communication, the two parties must exchange
+credentials with their identity information for mutual authentication purposes.
+On the client side, the server's identity is checked against the
+[secure naming](/docs/concepts/security/#secure-naming)
+information to see if it is an authorized runner of the workload. On the server
+side, the server can determine what information the client can access based on
+the
+[authorization policies](/docs/concepts/security/#authorization-policies),
+audit who accessed what at what time, charge clients based on the workloads they
+used, and reject any clients who failed to pay their bill from accessing the
+workloads.
+
+The Istio identity model uses the first-class `service identity` to
+determine the identity of a request's origin. This model allows for great
+flexibility and granularity for service identities to represent a human user, an
+individual workload, or a group of workloads. On platforms without a service
+identity, Istio can use other identities that can group workload
+instances, such as service names.
+
+The following list shows examples of service identities that you can use on different
+platforms:
+
+- Kubernetes: Kubernetes service account
+- GKE/GCE: GCP service account
+- GCP: GCP service account
+- AWS: AWS IAM user/role account
+- On-premises (non-Kubernetes): user account, custom service account,
+   service name, Istio service account, or GCP service account. The custom
+   service account refers to the existing service account just like the
+   identities that the customer's Identity Directory manages.
+
+## Public Key Infrastructure
```

Comment:
```suggestion
## Public Key Infrastructure (PKI) {#pki}
```

Reply:
I actually wonder if the parens should be reversed, since PKI is the commonly used term??
```suggestion
## PKI (Public Key Infrastructure) {#pki}
```